packages/react-bootstrap-table2/src/row-selection/wrapper.js

```diff
@@ -22,19 +22,17 @@ export default Base =>
       super(props);
       this.handleRowSelect = this.handleRowSelect.bind(this);
       this.handleAllRowsSelect = this.handleAllRowsSelect.bind(this);
-      props.store.selected = this.props.selectRow.selected || [];
+
+      props.store.selected = props.selectRow.selected || [];
       this.state = {
         selectedRowKeys: props.store.selected
       };
     }
 
     componentWillReceiveProps(nextProps) {
-      if (nextProps.selectRow) {
-        this.store.selected = nextProps.selectRow.selected || [];
-        this.setState(() => ({
-          selectedRowKeys: this.store.selected
-        }));
-      }
+      this.setState(() => ({
```

Comment:
Hi @AllenFang:

Remove null check of `nextProps.selectRow` because it was marked as `isRequired`.

One more thing is that I'm curious about the purpose of `componentWillReceiveProps`. So far we handled row selection by `handleRowSelect` and `handleAllRowsSelect`. Both of them would set `state.selectedRowKeys` in the end. Is it related to `remote` or something or maybe we don't need `componentWillReceiveProps`?